src/ServantElm.hs

```diff
@@ -172,7 +172,7 @@ elmPrimToDoc = \case
 
 mkTypeSignature :: Req ElmDefinition -> Doc
 mkTypeSignature request =
-  (hsep . punctuate " ->") ("String" : catMaybes [toMsgType, returnType])
+  (hsep . punctuate "->") ("String" : catMaybes [toMsgType, returnType])
```

Comment:
Did you verify that this doesn't cause an issue?

Reply:
it does cause an issue, I added the space back

Reply:
done

Reply:
Let's leave it like this for now, because I don't think it will be an issue, but in general, the "correct" way to do this would be something like
```haskell
let msgs :: [Doc ann]
let arr = "->"
hsep $ intersperse arr msgs
```
I think.

Reply:
Also, in general, when making a change, if it isn't too difficult, you should test out if your change breaks anything, before committing it (and try to fix things, if it does)